### PR TITLE
Intro revision and tile content clarification

### PR DESCRIPTION
**Pull request by ggetz**
@lilleyse This addresses a lot of the conceptual feedback we received, particularly on the distinction between a tile as a concept and the tile content as the binary which is an instance of a tile format. I also tried to add a little more context as to how the Batch Table and Feature Table fit in overall.

### CODE REVIEW

File: specification/README.md [hunk 1]
```diff
@@ -78,26 +78,30 @@ Acknowledgements:
 
 ## Introduction
 
-In 3D Tiles, a _tileset_ is a set of _tiles_ organized in a spatial data structure, the _tree_.  The tree incorporates the concept of Hierarchical Level of Detail (HLoD) for optimal rendering of spatial data.
+3D Tiles is designed for streaming and rendering massive 3D geospatial content such as Photogrammetry, 3D Buildings, BIM/CAD, Instanced Features, and Point Clouds. It defines a format for describing a hierarchical spatial data structure and a set of tile formats which deliver renderable 3D content. 3D Tiles does not define explicit rules for visualization of the content; a client may visualize 3D Tiles data however it sees fit.
```
Comment (lilleyse) [inline specification/README.md]: "It defines a format for describing a hierarchical spatial data" sounds wordy. Is "It defines a hierarchical..." enough?

File: specification/TileFormats/Batched3DModel/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ _Batched 3D Model_ allows offline batching of heterogeneous 3D models, such as d
 
 Per-model properties, such as IDs, enable individual models to be identified and updated at runtime, e.g., show/hide, highlight color, etc. Properties may be used, for example, to query a web service to access metadata, such as passing a building's ID to get its address. Or a property might be referenced on the fly for changing a model's appearance, e.g., changing highlight color based on a property value.
 
-A Batched 3D Model tile is a binary blob in little endian.
+An instance of a Batched 3D Model tile is a binary blob in little endian.
```
Comment (lilleyse) [inline specification/TileFormats/Batched3DModel/README.md]: I'm not really sure if it's necessary to say "an instance of". It sounds more confusing that way.

File: specification/TileFormats/Batched3DModel/README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ These semantics define global properties for all features.
 
 ## Batch Table
 
-The _Batch Table_ contains per-model application-specific metadata, indexable by `batchId`, that can be used for [declarative styling](../../Styling/README.md) and application-specific use cases such as populating a UI or issuing a REST API request.  In the binary glTF section, each vertex has an numeric `batchId` attribute in the integer range `[0, number of models in the batch - 1]`.  The `batchId` indicates the model to which the vertex belongs.  This allows models to be batched together and still be identifiable.
+The _Batch Table_ contains per-model application-specific metadata properties, indexable by `batchId`, that can be used for [declarative styling](../../Styling/README.md) and application-specific use cases such as populating a UI or issuing a REST API request.  In the binary glTF section, each vertex has an numeric `batchId` attribute in the integer range `[0, number of models in the batch - 1]`.  The `batchId` indicates the model to which the vertex belongs.  This allows models to be batched together and still be identifiable.
```
Comment (lilleyse) [inline specification/TileFormats/Batched3DModel/README.md]: "metadata properties" sounds verbose. Maybe just "properties".

File: specification/TileFormats/FeatureTable/README.md [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 ## Overview
 
-A _Feature Table_ describes position and appearance properties for each feature in a tile.  The [Batch Table](../BatchTable/README.md), on the other hand, contains per-feature application-specific metadata not necessarily used for rendering.
+A _Feature Table_ is a component of a tile's binary body and describes position and appearance properties required to render each feature in a tile. The [Batch Table](../BatchTable/README.md), on the other hand, contains per-feature application-specific metadata properties that a client uses to evaluate the declarative styling for a feature and may choose to use for further rendering or analysis.
```
Comment (lilleyse) [inline specification/TileFormats/FeatureTable/README.md]: The batch table counter-summary seems too detailed. The original wording was easier to follow. The change from metadata to properties is still good to keep though.

**Comment by ggetz** [comment]
Thanks @lilleyse, updated!

**Pull request merged**